src/App.tsx

```diff
@@ -2,108 +2,37 @@ import React from 'react';
 
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+import { PostList } from './components/PostList';
+import { Comment } from './Types/сomments';
+import { Post } from './Types/posts';
```

Comment:
You don’t have posts file, but you have Posts file
<img width="306" alt="Screenshot 2022-09-12 at 11 47 48" src="https://user-images.githubusercontent.com/47606379/189611633-057ae325-03c3-4846-a258-83f1c7fdd5b5.png">

(If you name files with upper first letter, use it always: all files should have names with a similar style - comments should rewrite to Comments)